test/acceptance/session_test.js

```diff
@@ -13,6 +13,35 @@ Scenario('simple session @WebDriverIO @Protractor @Puppeteer @Playwright', (I) =
   I.seeInCurrentUrl('/info');
 });
 
+Scenario('screenshots reflect the current page of current session @WebDriverIO @Protractor @Puppeteer @Playwright', async (I) => {
```

Comment:
is your fix covered all those helpers, if not just provide the one you covered like `@Puppeteer @Playwright`?

Reply:
Done!